.github/workflows/rollback-prod.yml

```
@@ -0,0 +1,37 @@
+          - batch
+        required: true
+
+jobs:
+  rollback:
+    concurrency:
+      group: prod-runtime
+      cancel-in-progress: false
+    uses: ./.github/workflows/_ansible-exec.yml
+    with:
+      environment_name: prod
+      ssh_host: ${{ secrets.PROD_SSH_HOST }}
```

actionlint (1.7.12)

[error] context "secrets" is not allowed here. available contexts are "github", "inputs", "matrix", "needs", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)